bundesliga_app/views.py

```diff
@@ -65,12 +65,16 @@ def post(self, *args, **kwargs):
         for event_in_api in events:
             # If the event id is an event of organizer
             if event_in_api['id'] in events_id:
+                logo_dict = event_in_api['logo']
+                logo = ""
+                if logo_dict:
+                    logo = event_in_api['logo']['url']
```

Comment:
i think this could be

> logo_dict = event_in_api['logo']
>                 if logo_dict:
>                     logo = event_in_api.get('logo').ge('url', "")

I'm not 100% sure maybe @eldalai can confirm this